terraform/locals.tf

```diff
@@ -86,6 +86,18 @@ locals {
     "egress",
   ]
 
+  # Get the public IPs associated with our nessus instances.
+  # Since our elastic IPs are handled differently in
+  # production vs. non-production workspaces, their corresponding
+  # Terraform resources (data.aws_eip.cyhy_nessus_eips,
+  # data.aws_eip.cyhy_nessus_random_eips) may or may not be created.  To
+  # handle that, we use coalescelist() to choose the (non-empty) list
+  # containing the valid eip objects.
+  nessus_public_ips = coalescelist(
+    data.aws_eip.cyhy_nessus_eips,
+    aws_eip.cyhy_nessus_random_eips,
+  )
```

Comment:
As long as we're making this a local variable, we should utilize it here too:
https://github.com/cisagov/cyhy_amis/blob/2d2a70bc3bc51cacce4e0babd17cec3bb5363d6f/terraform/cyhy_nessus_ec2.tf#L106-L109

Reply:
Please see [80d489a](https://github.com/cisagov/cyhy_amis/pull/425/commits/80d489a197e72d874000db8860f4581c65845767).

Reply:
I'm going to speak for @dav3r here (since he's out today) and say that commit 80d489a fully addresses his comment.

Reply:
Thanks @jsf9k for covering for me here.  You were indeed correct - this commit looks great to me.